From: "Guido S. Nickels" <gsn@kernel-oops.de>
Date: Sun, 5 Jun 2016 10:12:32 +0200
Subject: install target

set install target to /usr/bin instead of ${CMAKE_SOURCE_DIR}/deploy/bin
---
 CMakeLists.txt                       | 4 ++--
 src/hyperion-aml/CMakeLists.txt      | 2 +-
 src/hyperion-dispmanx/CMakeLists.txt | 2 +-
 src/hyperion-remote/CMakeLists.txt   | 2 +-
 src/hyperion-v4l2/CMakeLists.txt     | 2 +-
 src/hyperion-x11/CMakeLists.txt      | 2 +-
 src/hyperiond/CMakeLists.txt         | 2 +-
 7 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 487b066..adcf9e1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -63,8 +63,8 @@ if(ENABLE_OSX AND ENABLE_DISPMANX)
 endif(ENABLE_OSX AND ENABLE_DISPMANX)
 
 
-SET ( PROTOBUF_INSTALL_BIN_DIR ${CMAKE_BINARY_DIR}/proto )
-SET ( PROTOBUF_INSTALL_LIB_DIR ${CMAKE_BINARY_DIR}/proto )
+SET ( PROTOBUF_INSTALL_BIN_DIR /usr/share/hyperion/proto )
+SET ( PROTOBUF_INSTALL_LIB_DIR /usr/share/hyperion/proto )
 
 #if(ENABLE_QT5)
 # TODO vs ENABLE_QT4?
diff --git a/src/hyperion-aml/CMakeLists.txt b/src/hyperion-aml/CMakeLists.txt
index 9445b3e..6c206b7 100644
--- a/src/hyperion-aml/CMakeLists.txt
+++ b/src/hyperion-aml/CMakeLists.txt
@@ -61,4 +61,4 @@ else(ENABLE_QT5)
 	qt4_use_modules(${PROJECT_NAME} Core Gui Network )
 endif(ENABLE_QT5)
 
-install ( TARGETS ${PROJECT_NAME} DESTINATION "${CMAKE_SOURCE_DIR}/deploy/bin" )
+install ( TARGETS ${PROJECT_NAME} DESTINATION "/usr/bin" )
diff --git a/src/hyperion-dispmanx/CMakeLists.txt b/src/hyperion-dispmanx/CMakeLists.txt
index ae4c02d..966a68c 100644
--- a/src/hyperion-dispmanx/CMakeLists.txt
+++ b/src/hyperion-dispmanx/CMakeLists.txt
@@ -61,4 +61,4 @@ else(ENABLE_QT5)
 endif(ENABLE_QT5)
 
 
-install ( TARGETS ${PROJECT_NAME} DESTINATION "${CMAKE_SOURCE_DIR}/deploy/bin" )
+install ( TARGETS ${PROJECT_NAME} DESTINATION "/usr/bin" )
diff --git a/src/hyperion-remote/CMakeLists.txt b/src/hyperion-remote/CMakeLists.txt
index 36ca8cc..e17af02 100644
--- a/src/hyperion-remote/CMakeLists.txt
+++ b/src/hyperion-remote/CMakeLists.txt
@@ -42,5 +42,5 @@ else(ENABLE_QT5)
 	qt4_use_modules(${PROJECT_NAME} Core Gui Network )
 endif(ENABLE_QT5)
 
-install ( TARGETS ${PROJECT_NAME} DESTINATION "${CMAKE_SOURCE_DIR}/deploy/bin" )
+install ( TARGETS ${PROJECT_NAME} DESTINATION "/usr/bin" )
 
diff --git a/src/hyperion-v4l2/CMakeLists.txt b/src/hyperion-v4l2/CMakeLists.txt
index ee1882a..015e48d 100644
--- a/src/hyperion-v4l2/CMakeLists.txt
+++ b/src/hyperion-v4l2/CMakeLists.txt
@@ -61,4 +61,4 @@ else(ENABLE_QT5)
 endif(ENABLE_QT5)
 
 
-install ( TARGETS ${PROJECT_NAME} DESTINATION "${CMAKE_SOURCE_DIR}/deploy/bin" )
+install ( TARGETS ${PROJECT_NAME} DESTINATION "/usr/bin" )
diff --git a/src/hyperion-x11/CMakeLists.txt b/src/hyperion-x11/CMakeLists.txt
index 69744f4..df11c29 100644
--- a/src/hyperion-x11/CMakeLists.txt
+++ b/src/hyperion-x11/CMakeLists.txt
@@ -61,4 +61,4 @@ else(ENABLE_QT5)
 	qt4_use_modules(${PROJECT_NAME} Core Gui Network )
 endif(ENABLE_QT5)
 
-install ( TARGETS ${PROJECT_NAME} DESTINATION "${CMAKE_SOURCE_DIR}/deploy/bin" )
+install ( TARGETS ${PROJECT_NAME} DESTINATION "/usr/bin" )
diff --git a/src/hyperiond/CMakeLists.txt b/src/hyperiond/CMakeLists.txt
index 5aef74d..c8f4f40 100644
--- a/src/hyperiond/CMakeLists.txt
+++ b/src/hyperiond/CMakeLists.txt
@@ -34,4 +34,4 @@ if (ENABLE_PROTOBUF)
 	target_link_libraries(hyperiond protoserver)
 endif (ENABLE_PROTOBUF)
 
-install ( TARGETS hyperiond DESTINATION "${CMAKE_SOURCE_DIR}/deploy/bin" )
+install ( TARGETS hyperiond DESTINATION "/usr/bin" )
